doc/src_usermanual/techrep_main.tex

```diff
@@ -617,7 +617,7 @@ \subsubsection{The C++ interface}
 \begin{itemize} 
 \item first six: see \texttt{eval\_cons}.
 \item \texttt{nnzJacS}, \texttt{iJacS}, \texttt{jJacS}, \texttt{MJacS} are for number of nonzeros, $(i,j)$ indexes, and nonzero values of   the sparse Jacobian.
-\item  JacD: dense Jacobian as a contiguous array storing the matrix by rows; the array is ``primed'' to support double indexing \texttt{JacD[i][j]}.
+\item \texttt{JacD} should contain the Jacobian with respect to the dense variables of the MDS problem. The array should store this Jacobian submatrix row-wise, meaning that the each row of the Jacobian is contiguous in memory and starts right after the previous row.
 \end{itemize}
 
 Notes for implementer of this method: 

```

Comment:
Should we use \warningcp{Note:}? like line 643

Reply:
done